openlibrary/plugins/worksearch/code.py

```diff
@@ -1272,7 +1224,7 @@ def work_search(
             facet=facet,
             spellcheck_count=spellcheck_count,
         )
-        response = json.loads(reply)['response'] or ''
+        response = reply['response'] or ''
```

Comment:
`None` case is not handled here.  This is causing my local instance's homepage to error on first load, resulting in no content (aside from the page header and footer).

Reply:
This sort of worked:
`response = reply.get('response', '') if reply else ''`

The issue with this is that `response` is being subscripted like a `dict` later in this function, leading to more errors.

@cdrini, what are sensible defaults for `response.get('numFound')` and `response.get('docs')`?  Using `0` and `{}` respectively causes the homepage to render locally without the classics carousel:

![Screenshot from 2022-05-31 19-27-20](https://user-images.githubusercontent.com/28732543/171299983-3cc304c9-73d8-4ce3-b888-b1617e837542.png)

Reply:
#6616